<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmoothSetDlg</class>
 <widget class="QDialog" name="SmoothSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="noSmoothBtn">
     <property name="text">
      <string>NoSmppoth</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="oneThirdBtn">
     <property name="text">
      <string>1/3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="onesixthBtn">
     <property name="text">
      <string>1/6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="onetwelfthBtn">
     <property name="text">
      <string>1/12</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="oneTwenty_fourthBtn">
     <property name="text">
      <string>1/24</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="oneForty_eighthBtn">
     <property name="text">
      <string>1/48</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
